provision/testdata/flavor_localhost.inp.yaml

```diff
@@ -28,13 +28,18 @@ aci_config:
 net_config:
   node_subnet: 1.100.202.1/24              # Subnet to use for nodes
   pod_subnet: 10.2.56.1/21 # Subnet to use for Kubernetes Pods
+  gbp_pod_subnet: 10.3.56.1/21
```

Comment:
Please derive the gbp_pod_subnet in gw/subnet_length form using the cni gateway information and the pod subnet network length, as discussed.